README.md

```diff
@@ -140,10 +140,10 @@ from GitHub. You may make your own fork project from our project.
 ### 1. Debian/Ubuntu
 
 - gcc
-- libncurses-dev
+- libncurses5-dev

```

Comment:
Why libncurses5-dev? libncurses-dev is a meta package and it refers to libncurses5-dev